# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../udev/runpatch.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- udev-182/src/libudev.c.vanilla	2020-03-08 14:27:48.536223015 +0100
+++ udev-182/src/libudev.c	2020-03-08 14:28:48.600218073 +0100
@@ -251,7 +251,7 @@
                         goto err;
 
         if (udev->run_path == NULL)
-                if (set_value(&udev->run_path, "/run/udev") == NULL)
+                if (set_value(&udev->run_path, "/dev/.udev") == NULL)
                         goto err;
 
         if (udev->rules_path[0] == NULL) {
@@ -426,7 +426,7 @@
  * udev_get_run_path:
  * @udev: udev library context
  *
- * Retrieve the udev runtime directory path. The default is "/run/udev".
+ * Retrieve the udev runtime directory path. The default is "/dev/.udev".
  *
  * Returns: the runtime directory path
  **/
